<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PLCData</class>
 <widget class="QWidget" name="PLCData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PLCData</string>
  </property>
  <widget class="QPushButton" name="BT_OpenPLC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>打开PLC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_ClosePLC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭PLC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_DeviceName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>软元件名称</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_ReadPLC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>读取PLC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_WritePLC">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>写入PLC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit_WritePLCValue">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>41</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textshowReadPLC">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_ReadPLC_Block">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>批量读取PLC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_WritePLCBlock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>批量写入PLC</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>41</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string>读取个数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>41</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string>写入个数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_readplcNUM">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>170</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_writeplcNUM">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>240</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
